content/en/blog/2025/go-2025-goals.md

```diff
@@ -0,0 +1,129 @@
+
+Semantic conventions are foundational to OpenTelemetry and the cornerstone of
+data quality across the ecosystem. The OpenTelemetry community has recently
+updated the tooling used to generate these conventions into usable code by
+introducing the [weaver](https://github.com/open-telemetry/weaver) project. We
+plan to integrate this new tooling into the OpenTelemetry Go project and provide
+updates to the latest versions of semantic conventions.
+
+### SDK Self-Observability Signals
+
+- Priority: Yearly goal
+- Tracking Issue:
+  [#2547](https://github.com/open-telemetry/opentelemetry-go/issues/2547)
+- Sponsor: [@dashpole](https://github.com/dashpole)
+
+This goal aims to enhance the observability of the OpenTelemetry Go SDK itself.
+We plan to add metrics about the tracing portions of the SDK as a first step,
+but hope to expand this with more signals measuring all areas of the SDK.
+Unified semantic conventions across all OpenTelemetry languages will play a
+critical role in achieving this objective.
+
+### Go Runtime Metrics Stabilization
+
+- Priority: Yearly goal
+- Tracking Issue:
+  [#5655](https://github.com/open-telemetry/opentelemetry-go-contrib/issues/5655)
+- Sponsor: [@dashpole](https://github.com/dashpole)
+
+Recently, the Go team
+[updated runtime metrics within the Go language](https://github.com/golang/go/issues/67120).
+These updates have been
+[codified in OpenTelemetry semantic conventions](https://github.com/open-telemetry/semantic-conventions/pull/981),
+and are provided as opt-in metrics in the
+[`runtime` package](https://pkg.go.dev/go.opentelemetry.io/contrib/instrumentation/runtime#pkg-overview).
+The Go SIG plans to gather community feedback and transition these metrics to an
+opt-out model, allowing better observability of Go runtimes.
+
+### Logs API Stability
+
+- Priority: Yearly goal
+- Tracking Project:
+  [Go: Logs (GA)](https://github.com/orgs/open-telemetry/projects/43)
+- Sponsor: [@pellared](https://github.com/pellared)
+
+Stabilizing the Logs API is crucial for providing a logging solution that aligns
+with OpenTelemetry’s overarching goals. Currently, a non-stable "beta"
+implementation of this API is provided in the
+[`log` package](https://pkg.go.dev/go.opentelemetry.io/otel/log), along with
+many bridges to popular logging packages:
+
+- [`otellogr`](https://pkg.go.dev/go.opentelemetry.io/contrib/bridges/otellogr)
+- [`otellogrus`](https://pkg.go.dev/go.opentelemetry.io/contrib/bridges/otellogrus)
+- [`otelslog`](https://pkg.go.dev/go.opentelemetry.io/contrib/bridges/otelslog)
+- [`otelzap`](https://pkg.go.dev/go.opentelemetry.io/contrib/bridges/otelzap)
+- [`otelzerolog`](https://pkg.go.dev/go.opentelemetry.io/contrib/bridges/otelzerolog)
+
+The Go SIG plans to continue its effort in developing the upstream
+specification. Work to stabilize the OpenTelemetry Go implementation depends on
+this upstream development, including the development of the event signal.
```

Comment:
nit: I do not think that "Events" are a separate signal. I think that calling it like that can cause confusion.

```suggestion
this upstream development, including the addition of Events.
```